cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/SortedReplicas.java

```diff
@@ -81,29 +84,26 @@
   }
 
   /**
-   * Get the sorted replicas in the ascending order of their priority and score.
+   * Get the sorted replica wrappers in the ascending order of their priority and score.
    * This method initialize the sorted replicas if it hasn't been initialized.
    *
-   * @return the sorted replicas in the ascending order of their priority and score.
+   * @return the sorted replicas wrappers in the ascending order of their priority and score.
    */
-  public NavigableSet<ReplicaWrapper> sortedReplicas() {
+  public NavigableSet<ReplicaWrapper> sortedReplicaWrappers() {
     ensureInitialize();
     return Collections.unmodifiableNavigableSet(_sortedReplicas);
   }
 
   /**
-   * Get a list of replica wrappers in the descending order of their priority and score.
+   * Get the sorted replicas in the ascending order of their priority and score.
    * This method initialize the sorted replicas if it hasn't been initialized.
    *
-   * @return a list of replica wrappers in the descending order of their priority and score.
+   * @return the sorted replicas in the ascending order of their priority and score.
    */
-  public List<ReplicaWrapper> reverselySortedReplicaWrappers() {
```

Comment:
I am curious why we removed `reverselySortedReplicaWrappers` -- is it because it is currently unused and expected to be unused?

Reply:
yes, I feel whenever we want sorted replica list, we should use 
`public List<Replica> sortedReplicas()` and 
`public List<Replica> reverselySortedReplicas()`
Actually I feel `ReplicaWrapper` should just be an inner helper Class.